### PR TITLE
Auto-start ngi_pipeline with ssh and curl

### PR DESCRIPTION
**Pull request by johandahlberg**
This is just an untested quick hack - but I wanted the opinion of you guys if you think that this is a viable way to auto-start the `ngi_pipeline` for now. I'm thinking that this could be a fast solution that we could use now until we have Hercules fully operational. It has the benefit that it will work also with starting Sisyphus manually - which we would miss if we incorporated it into to Hercules right now.

### CODE REVIEW

File: sisyphus.pl [hunk 1]
```diff
@@ -1097,6 +1109,42 @@ =head1 DESCRIPTION
...
+    
+    if [ -z "\$3" ]
+    then
+      echo "-runfolder name path #3 is zero length or empty."
+      exit 1;
+    else
+       runfolderName=\$3;
+    fi
+    
+    ssh \$remoteHost "curl \$remoteHost:\$remotePort/flowcell_analysis/runfolderName"
```
Comment (b97pla) [inline sisyphus.pl]: should the host in the curl command perhaps be localhost or will it resolve to that on the remote anyway?

**Comment by b97pla** [comment]
I think this is a good suggestion and a rather simple and clean implementation, so not much that would need maintaining or fiddling with. I'm voting :+1:

**Comment by johandahlberg** [comment]
@b97pla I've finished this now - as far as I can see it should be working (though that of course needs to be properly validated). I also added syntax checking to Travis. I'm thinking that you could review this here, and then when you're done I can close this PR and open a new one with some rebasing first so that the commit log doesn't look so bloated.

File: .travis.yml [hunk 2]
```diff
@@ -3,13 +3,18 @@ perl:
...
+  - sudo apt-get install -y libxml-libxslt-perl
+  - sudo apt-get install -y libgdbm3  
   - "cpanm --quiet --notest PerlIO::gzip"
+  - "cpanm --quiet --notest Digest::MD5"
   - "cpanm --quiet --notest XML::Simple"
   - "cpanm --quiet --notest Archive::Zip"
   - "cpanm --quiet --notest PDL"
-  - sudo apt-get update -qq
-  - sudo apt-get install -y dos2unix
+  - "cpanm --quite --notest XML::LibXSLT"
```
Comment (b97pla) [inline .travis.yml]: should be --quiet?

Comment (johandahlberg) [inline .travis.yml]: Who should be quite? :scream_cat:

Comment (b97pla) [inline .travis.yml]: quite quiet

**Comment by b97pla** [comment]
Ok, looks good, do the rebasing and pull request again and I'll merge

**Pull request closed without merging**